plugins/GSdx/Renderers/Vulkan/vulkan.inl

```diff
@@ -0,0 +1,70 @@
+#ifdef VK_GLOBAL_FUNC
+VK_GLOBAL_FUNC( vkGetInstanceProcAddr )
```

Comment:
This is all we need for now. Keep the ifdefs for the others but remove the VK_*_FUNC parts for the rest. We will add them as we need them.

Reply:
I stashed what we had before in case we need it again.